scripts/deploy-contracts-genesis.sh

```diff
@@ -122,6 +131,6 @@ _add_wasm_message() {
     local -r init_msg="$8"
 
     run_cmd "$genesis_home_dir" add-wasm-genesis-message instantiate-contract "$code_id" "$init_msg" --label "$contract_name" \
-      --run-as "$admin_addr" --no-admin $amount_flag
+      --run-as "$admin_addr" --admin "$(admin_contract_instance_addr)" $amount_flag
```

Comment:
We do not need yet another admin, so let's unify them into being only the Admin contract. Please review the `internal/genesis.sh::generate_genesis()` from `nolus-core` for more info on how we call this code.